src/widget/wwidgetstack.cpp

```diff
@@ -114,6 +156,7 @@ void WWidgetStack::addWidgetWithControl(QWidget* pWidget, ControlObject* pContro
                                         int on_hide_select) {
     int index = addWidget(pWidget);
     if (pControl) {
+        m_triggers[index] = pControl;
```

Comment:
pControl may not live as long as the widgetstack so you should make a COSlave of its configkey -- or make m_triggers a map of the WidgetStackControlListeners and get access to the COThread member variable they have